### PR TITLE
(CDAP-1604) Don't enable explore on system datasets

### PR DESCRIPTION
**Pull request by ParthGandhi**
https://issues.cask.co/browse/CDAP-1604

### CODE REVIEW

**Comment by ParthGandhi** [comment]
http://builds.cask.co/browse/CDAP-DUT1129-1/

File: cdap-explore-client/src/main/java/co/cask/cdap/explore/client/ExploreFacade.java [hunk 1]
```diff
@@ -144,6 +144,10 @@ public void removeNamespace(Id.Namespace namespace) throws ExploreException, SQL
     handleExploreFuture(futureSuccess, "remove", "namespace", namespace.getId());
   }
 
+  private boolean isDatasetExplorable(Id.DatasetInstance datasetInstance) {
```
Comment (bdmogal) [inline cdap-explore-client/src/main/java/co/cask/cdap/explore/client/ExploreFacade.java]: nit: The contract of this method seems to indicate that it operates on a dataset and returns if it is explorable. To check if explore is enabled in CDAP as a whole seems to be outside of the purview of this method. 

Only a nit though.

Comment (ParthGandhi) [inline cdap-explore-client/src/main/java/co/cask/cdap/explore/client/ExploreFacade.java]: Good point. Fixed.

**Comment by ParthGandhi** [comment]
New build: http://builds.cask.co/browse/CDAP-DUT1129-2/commit

**Comment by bdmogal** [comment]
:+1: Wish there was a way to add a unit test for this, I can't think of anything right now, maybe we can add something later. Let's just keep an eye open for the stack trace again.

**Pull request merged**